plugins/legacy-plugin-chart-event-flow/src/controlPanel.tsx

```diff
@@ -89,7 +89,7 @@ const config: ControlPanelConfig = {
       controlSetRows: [['adhoc_filters']],
     },
     {
-      label: t('Additional metadata'),
+      label: t('Metadata'),
```

Comment:
![image](https://user-images.githubusercontent.com/812905/99365793-98332600-286c-11eb-893b-6bb0d871d198.png)
 (this is admittedly a bit redundant with the matching headers)

Reply:
what is the metadata used for? Maybe we can make the title even more explanatory?

Reply:
hey Evan, we shouldn't remove "Additional" here, adding an addition meta data is not gonna affect the chart/query itself, it's just adding an additional column to the table generated by the query... so, let's not remove 'additional'. but we should have a better tooltip for sure.. also, please keep the Event definition section open by default, right now, it's closed by default. 
Chris W made this chart, it can be very useful for monitoring event status change but not many people are using it or understand how to use it.... 
<img width="867" alt="Screen Shot 2020-11-18 at 5 55 48 PM" src="https://user-images.githubusercontent.com/67837651/99610988-af3a5b00-29c7-11eb-881d-e2279b904d36.png">

@ktmud any thoughts?

Reply:
Let's keep the "Additional metadata" section title for now. In the future, things like this should probably be in a "Tooltip" section in chart customization.